### PR TITLE
feat: display of user location using LocationManager

### LINKED ISSUE #1386: Getting user location for Map Activity
**Actual Behaviour**

Right now the Map Activity doesnt have useful features which can provide more interaction with users.

**Expected Behaviour**

Currently I want to get the users location to use that ahead for many applications in Map Activity.

**Steps to reproduce it**

-

**LogCat for the issue**

-

**Screenshots of the issue**

-

**Would you like to work on the issue?**

Yes.

@the-dagger 
please review this. and also instead of using the variant i think in this case i have to use the google play services plugin which requires a google-services.json file. How to go about this file so that i can work with my location?

### PR DESCRIPTION
**Pull request by arp95**
Fixes #1386 

Changes: 

Used Location Manager to display user location which checks for network as well as gps connection for finding the users current location.

Screenshots for the change: 

-

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 1]
```diff
@@ -217,8 +306,13 @@ protected void onCreate(Bundle savedInstanceState) {
         setUpProgressBar();
         setUpCustomTab();
 
+        getLocation();
         disposable = new CompositeDisposable();
 
+        if(latitude!=0.0 && longitude!=0.0){
+            Toast.makeText(this, "Location is " + latitude + " " + longitude, Toast.LENGTH_LONG).show();
```
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Why are you displaying a toast to the user?

**Comment by arp95** [comment]
@the-dagger
right now i wanted to get the method of taking location approved. now i plan on using the location in different use cases if approved.

**Comment by the-dagger** [comment]
@arp95 
Closing this for now, we can probably follow up on this later on.
Right now, getting user location isn't a top priority so we can defer it for later.

**Comment by arp95** [comment]
@the-dagger 
sure! No problem.

**Pull request closed without merging**